Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackTests.swift

```diff
@@ -928,6 +928,22 @@ class AVFoundationPlaybackTests: QuickSpec {
                 }
             }
 
+            context("when onFailedToPlayToEndTime") {
+                it("dispatch error") {
+                    let playback = AVFoundationPlaybackMock(options: [:])
+                    let notification = NSNotification(name: NSNotification.Name(""), object: playback.player?.currentItem)
+                    var didErrorCalled = false
+
+                    playback.onFailedToPlayToEndTime(notification: notification)
+
+                    playback.on(Event.error.rawValue) { _ in
+                        didErrorCalled = true
+                    }
+
+                    expect(didErrorCalled).to(beFalse())
```

Comment:
Maybe we should test if the error called is true, for this we should set the callback before fires the notification, what you think about that?

Reply:
Done